baremetal/metal3data_manager.go

```diff
@@ -659,8 +659,8 @@ func (m *DataManager) releaseAddressesFromPool(ctx context.Context,
 	return nil
 }
 
-// getAddressFromPool adds an ownerReference on the referenced Metal3IPPool
-// objects. It then tries to fetch the Metal3IPAddress if it exists, and asks
+// getAddressFromPool creates an IPClaim if it does not exist.
+// It then tries to fetch the Metal3IPAddress if it exists, and asks
```

Comment:
A bit confused about removing Metal3 prefix from IPClaim in some comments while leaving for others, for example here we are leaving the prefix for Metal3IPAddress

Reply:
Oh I see. I was mostly changing the comments that were confusing (mentioned IPPool instead of IPClaim) and didn't think about the prefix. Should I add it back? Or better to remove it everywhere?

Reply:
I would let the prefix be everywhere to be specific